pkg/sriov/sriov.go

```diff
@@ -192,7 +107,11 @@ func (s *sriovManager) SetupVF(conf *sriovtypes.NetConf, podifName string, cid s
 			return fmt.Errorf("error setting container interface name %s for %s", linkName, tempName)
 		}
 
-		// 6. Bring IF up in Pod netns
+		// 6. Enable IPv4 ARP notify and IPv6 Network Discovery notify
+		// Error is ignored here because enabling this feature is only a performance enhancement.
+		_ = s.utils.EnableArpAndNdiscNotify(podifName)
```

Comment:
question should we return error here and continue to skip the error on `announceIPs`?

Reply:
@SchSeba this is regarded as performance enhancement and so i would not want it to fail if not needed.

Reply:
Yes we agreed that any function that is part of any performance enhancement shall not fail. Resolving this comment.